app/js/components/LabResultEntry.jsx

```diff
@@ -135,15 +136,20 @@ export class LabResultEntry extends PureComponent {
             <div className="did-not-perform">
               <span className="did-not-perform-label">Reason:&nbsp;</span>
               <Obs
-                conceptAnswers={["Low Volume",
+                conceptAnswers={[" ",
+                  "Low Volume",
                   "Contaminated",
                   "Equipment Failure",
                   "Stock Out",
                   "Blood Coagulation",
                   "Inappropriate Container"]}
                 widget="dropdown"
-                concept={CONSTANTS.labResultsDidNotPerformReason}
-                path="did-not-perform-dropdown"
+                concept={CONSTANTS.labResultsDidNotPerformQuestion}
+                path="did-not-perform-reason"
+                handleSelect={(field, selected) => {
+                  dispatch(change(formKey, `obs|path=did-not-perform-checkbox|concept=${CONSTANTS.labResultsDidNotPerformQuestion}`, selected));
```

Comment:
Why do we need to dispatch a change event in these cases?

Reply:
Each time the user toggles the dropdown, we update the value of the checkbox, to be same with the value of the dropdown.